web/styles/popovers.css

```diff
@@ -598,7 +598,8 @@ ul.popover-group-menu-member-list {
         }
 
         .stream-privacy {
-            display: inline-block;
+            display: flex;
+            align-items: center;
```

Comment:
For this commit message, it's also be very helpful to be specific; "the stream list" could refer to a lot of things, most of which are not popovers. I would use `popovers: ` in the commit message, and have the body mention specifically which component you're fixing.

Reply:
I have made the changes. Please review the updates and let me know if there’s anything else I can improve.